support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/endpoints/OAuth20AuthorizeEndpointController.java

```diff
@@ -62,9 +63,13 @@ public ModelAndView handleRequest(final HttpServletRequest request,
         val context = new JEEContext(request, response, getOAuthConfigurationContext().getSessionStore());
         val manager = new ProfileManager<CommonProfile>(context, context.getSessionStore());
 
-        if (!verifyAuthorizeRequest(context) || !isRequestAuthenticated(manager)) {
-            LOGGER.error("Authorize request verification failed. Authorization request is missing required parameters, "
-                + "or the request is not authenticated and contains no authenticated profile/principal.");
+        if (context.getRequestAttribute(OAuth20Constants.ERROR).isPresent()) {
+            LOGGER.error("Authorize request verification failed");
```

Comment:
This is not helpful. You need to, if you really want to, explain exactly what failed and how; the message here is very generic and it confuses more than it helps. If you don't have enough information to explain things, remove the line, specially if the error is tracked in logs via some other component.

Reply:
Ok. I'll remove it because the other components are logging all the errors.

Reply:
Done